ssl-monitor/api/main.py

```
@@ -650,6 +650,230 @@ async def add_domain(
         )
 
 
+@app.delete("/api/domains/{domain}", summary="Delete a domain from monitoring")
+async def delete_domain(
+    domain: str,
+    db: Session = Depends(get_db),
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)